mypy/errors.py

```diff
@@ -310,7 +310,11 @@ def report(self,
         self.add_error_info(info)
 
     def _add_error_info(self, file: str, info: ErrorInfo) -> None:
-        assert file not in self.flushed_files
+        if file in self.flushed_files:
+            self.report(info.line, None,
```

Comment:
I copied the basic format for the warning from https://github.com/python/mypy/blob/00530dbc09e8b9a19bf1279e9ec7a6a0cd40cc76/mypyc/genops.py#L122

Note that `only_once=True` is necessary to prevent recursing infinitely.